src/components/HomeScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import { NavigationProp } from '@react-navigation/native';
+import React from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+} from 'react-native';
+
+import GlobalStyles from '../GlobalStyles';
+
+import Button from './shared/Button';
+
+interface Props {
+  navigation: NavigationProp<any>
```

Comment:
If we're using TS, I would think we type most things. For this, you can see here: https://reactnavigation.org/docs/typescript/#type-checking-screens 
Happy to wire this up as well in future PRs.

Reply:
I'm sure this will be helpful :+1: